packages/terra-framework-docs/src/terra-dev-site/doc/worklist-data-grid/example/DefaultWorklistDataGrid.jsx

```diff
@@ -1,22 +1,98 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import WorklistDataGrid from 'terra-worklist-data-grid';
 import gridDataJSON from './gridData.json';
 
 const DefaultWorklistDataGrid = () => {
+  const rowSelectionModeRef = useRef();
   const rowHeaderIndex = 0;
   const { cols, rows } = gridDataJSON;
+  const [selectedRows, setSelectedRows] = useState([]);
```

Comment:
The WorklistDataGrid should not be maintaining the selected rows in its state.  The consumer would be managing the list of selected rows.